src/service/resources-impl.js

```diff
@@ -876,6 +901,21 @@ export class Resources {
       }
     }
 
+    // collect resources that are in the first viewport
+    // for instrumentation.
+    if (this.shouldFindFirstViewportElements_) {
+      for (let i = 0; i < this.resources_.length; i++) {
+        const r = this.resources_[i];
+        if (r.hasOwner()) {
+          continue;
+        }
+        if (r.isDisplayed() && r.overlaps(viewportRect)) {
+          this.firstViewportElements_.push(r);
+        }
+      }
+      this.shouldFindFirstViewportElements_ = this.firstPassAfterDocumentReady_;
```

Comment:
Don't understand this statement. Can't find `firstPassAfterDocumentReady_` variable anywhere.